app/models/pawn.rb

```diff
@@ -1,2 +1,28 @@
 class Pawn < Piece
+
+    INITIAL_WHITE_PAWN_LOCATIONS = [[x0, y0], [x1, y1], [x2, y2], [x3, y3], [x4, y4], [x5, y5], [x6, y6] [x7, y7]]
+    INITIAL_BLACK_PAWN_LOCATIONS = [[x0, y0], [x1, y1], [x2, y2], [x3, y3], [x4, y4], [x5, y5], [x6, y6] [x7, y7]]
+
+    def initial_pawn_locations
+        if self.color == 'white'
+          INITIAL_WHITE_PAWN_LOCATIONS.include?(self.x_position, self.y_position)
+        else
+          INITIAL_BLACK_PAWN_LOCATIONS.include?(self.x_position, self.y_position)
+        end
+    end
```

Comment:
hint:

Try this out in a console.
```Ruby
positions =  [[0,1]] # notice - these have to be integers
positions.include? [0,1]
positions.include? [1,1]

```

Doesn't need to be a rails console. Using `irb` or `pry` will work as well.